<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>531</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSub_system">
    <property name="title">
     <string>sub_system</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionCase"/>
    <addaction name="actionUser"/>
    <addaction name="actionWorkflow"/>
    <addaction name="actionWarning"/>
   </widget>
   <addaction name="menuSub_system"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCase"/>
   <addaction name="actionUser"/>
   <addaction name="actionWorkflow"/>
   <addaction name="actionWarning"/>
  </widget>
  <action name="actionLogin">
   <property name="icon">
    <iconset>
     <activeon>:/icons/task_online.png</activeon>
    </iconset>
   </property>
   <property name="text">
    <string>login</string>
   </property>
  </action>
  <action name="actionCase">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/case.png</normaloff>:/icons/case.png</iconset>
   </property>
   <property name="text">
    <string>case</string>
   </property>
  </action>
  <action name="actionUser">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/my.png</normaloff>:/icons/my.png</iconset>
   </property>
   <property name="text">
    <string>user</string>
   </property>
  </action>
  <action name="actionWorkflow">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/workflow02.png</normaloff>:/icons/workflow02.png</iconset>
   </property>
   <property name="text">
    <string>workflow</string>
   </property>
  </action>
  <action name="actionWarning">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/warningcolor02.png</normaloff>:/icons/warningcolor02.png</iconset>
   </property>
   <property name="text">
    <string>warning</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
